customerio-reactnative.podspec

```diff
@@ -19,6 +19,10 @@ Pod::Spec.new do |s|
 
   # Syntax of native iOS pods allows for automatically upgrading to the latest major version of the iOS SDK. 
   # Reference: https://guides.cocoapods.org/syntax/podfile.html#pod
-  s.dependency "CustomerIO/Tracking", '~> 2'
-  s.dependency "CustomerIO/MessagingInApp", '~> 2'
+  s.dependency "CustomerIO/Tracking", package["cioNativeiOSSdkVersion"]
+  s.dependency "CustomerIO/MessagingInApp", package["cioNativeiOSSdkVersion"]
+
+  # Including APN push since there is no 3rd party dependencies required for it to work. 
+  # If customer uses FCM, they will use javascript functions included in SDK along with a 3rd party RN FCM SDK. 
+  s.dependency "CustomerIO/MessagingPushAPN", package["cioNativeiOSSdkVersion"]
```

Comment:
hmm, so folks who aren't even using `APN` would still have this dependency installed for them? 

>  If customer uses FCM, they will use javascript functions included in SDK along with a 3rd party RN FCM SDK.

Also, regarding FCM, I think we are taking an opinionated approach here, we need to validate and discuss this among squad if there are any possible issues that we expect here?